scripts/build-process/generate-pages-from-markdown.js

```diff
@@ -139,7 +146,7 @@ const routesHeader = fs.readFileSync(
 );
 fs.writeFileSync(
   PATH_TO_ROUTES_FILE,
-  `${routesHeader}\n${routesString.join('\n')}`,
+  `${routesHeader}\n${importsString}\n\n${routesString.join('\n')}`,
 );
 
 console.log(colors.green(`Updated ${PATH_TO_ROUTES_FILE}`));
```

Comment:
The build script now adds import statements for the markdown files instead of inlining them. The changes here enable this. We do this so that the markdown files can be targeted by webpack. Before they were trapped in strings so webpack couldn't get to them.